### PR TITLE
Fix tag language ignored when importing BibTex and upgrade gradle and v8

### PR DESCRIPTION
**Pull request by gallardo**

### CODE REVIEW

**Comment by gallardo** [comment]
:open_mouth: Travis doesn't support Java 8 anymore?

**Comment by gallardo** [comment]
There is still one test failing. See https://github.com/michel-kraemer/citeproc-java/issues/59

**Comment by kylase** [comment]
@michel-kraemer is it possible to merge this PR and release it?

**Comment by michel-kraemer** [comment]
I'm working on it.

**Comment by michel-kraemer** [comment]
@gallardo Thanks for the hard work. I'm currently going through each commit and try to merge it as carefully as possible. It would have been better though if you created more than one pull request. Do you think it's possible to split it?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 language: java
 jdk:
-  - oraclejdk8
```
Comment (michel-kraemer) [inline .travis.yml]: I merged this commit. Please rebase.

File: build.gradle [hunk 2]
```diff
@@ -3,7 +3,6 @@ subprojects {
     apply plugin: 'osgi'
     apply plugin: 'eclipse'
     
-    version = '1.1.0-SNAPSHOT'
```
Comment (michel-kraemer) [inline build.gradle]: Is there a specific reason for this commit? Please elaborate.

File: build.gradle [hunk 3]
```diff
@@ -41,8 +40,3 @@ subprojects {
         classifier = 'sources'
     }
 }
-
-// initialize gradle wrapper
```
Comment (michel-kraemer) [inline build.gradle]: I merged this. Please rebase.

**Comment by gallardo** [comment]
> @gallardo Thanks for the hard work. I'm currently going through each commit and try to merge it as carefully as possible. It would have been better though if you created more than one pull request. Do you think it's possible to split it?

I'm sorry about the not splitting. That was unintended. I created the PR from my master, and didn't realize that further updates into my master would also commit into the PR. I'll try to split it.

**Comment by michel-kraemer** [comment]
Thank you very much for the great work. 👍 I cherry picked some of the commits already. This hopefully makes it easier for you to trim down the pull request.

File: build.gradle (hunk 2 above)
Comment (gallardo) [reply build.gradle]: For convenience, I wanted to override versioning from the command line. E.g.:

`sh "./gradlew -Pversion=$VersionTag citeproc-java:jar"`

where `$VersionTag` includes the git branch number and the build number.

I'm using Jenkins internally to build the project. My versioning includes the BUILD_NUMBER. I could overwrite the version number in the `build.gradle` every build, but I prefer to set the versioning in the command line. This leads to less commits and is less error prone. The build still reads the version info from gradle.properties, so:
- for local builds (no `version` override) I get snapshots,
- and for jenkins builds, I get git branch versioning plus build info.

The easiest way I found to achieve this was extracting the versioning info out of the `build.gradle` into the `gradle.properties`. (See https://stackoverflow.com/a/40239092/413020.)

**Comment by gallardo** [comment]
I'm closing the PR. I have to review it and split it.

File: build.gradle (hunk 2 above)
Comment (gallardo) [reply build.gradle]: I have created https://github.com/michel-kraemer/citeproc-java/pull/68

File: build.gradle (hunk 3 above)
Comment (gallardo) [reply build.gradle]: Thanks

File: .travis.yml (hunk 1 above)
Comment (gallardo) [reply .travis.yml]: :+1:

**Pull request closed without merging**